pkg/sync/sync.go

```diff
@@ -556,11 +555,7 @@ func worker(tasks <-chan object.Object, src, dst object.ObjectStorage, config *C
 			}
 			var err error
 			if config.Links && obj.IsSymlink() {
-				if err = copyLink(src, dst, key); err == nil {
-					copied.Increment()
-					break
-				}
-				logger.Errorf("copy link failed: %s", err)
+				err = copyLink(src, dst, key)
```

Comment:
checkSum() doe not work with symlinks, we should skip it or fix it

Reply:
skipped

Reply:
Is there any idea to check the copied symlink?